### DISCUSSION TITLE
Podman support

### DISCUSSION THREAD

**Discussion by gaurav-ravi** [open-metadata/OpenMetadata#7358] [Q&A] [answered]
Hello! I was looking at the QuickStart instructions to use containers but it appears docker is tightly-coupled with the open metadata app. Is there any support planned for other container engines like Podman? If not, is there any workaround that I can use?

**Comment by harshach** [comment]
@gaurav-ravi we support docker and helm charts. 
@akash-jain-10 is there any easy way to support Podman

**Comment by akash-jain-10** [comment, marked as answer]
Hello @gaurav-ravi .
We haven't worked with Podman, but as per the description [here](https://podman.io/), it looks similar to Docker (like a container engine).

In the docs, we support docker-compose. After a little research, it seems Podman also has a similar flavour of [podman-compose](https://github.com/containers/podman-compose). 

Would you like to give that a try ?

Download docker-compose.yaml from Github Release 0.12.0 - https://github.com/open-metadata/OpenMetadata/releases/tag/0.12.0-release

Then, run `podman-compose up -d` in the same directory where docker-compose file is available.

**Comment by gaurav-ravi** [comment]
Hej Guys! many thanks for the response. That's right, I have been experimenting with podman-compose. Unfortunately I can't use the metadata cli. Anyway, It might be interesting to know there are some limitations with podman-compose 1.0.3  - while identifying certain network attributes in the yaml file. There is a way to get around it though. Apparently podman-compose 1.0.4 gets the containers running all the way. But this is a developer version as of this writing - hopefully a stable version will be out soon. 

Also, it did help me to find docker-compose-postgres.yml in OpenMetadata 0.12.0-Release. I couldn't get the containers to connect with Mysql.
